zkvm-prover/build-guest.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
will we build guest inside this repo or still use zkvm-prover?
need this script here?